wordpress-mobile/WordPress-Android
# Removes: un-used resources using the Android studio tool 1/2

Fixes #
## Description
This PR deletes the unused resources in the code using [Remove Unused Resources tool ](https://stackoverflow.com/questions/6373482/remove-all-unused-resources-from-an-android-project)in the Android Studio


## To Test:
1. Smoke test the JP And WP app and verify that everything is working as expected 
2. Verify that the classess, resources and files deleted are unused. 

## Regression Notes

1. Potential unintended areas of impact
App is not working as expected

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and Smoke testing 

3. What automated tests I added (or what prevented me from doing so)
Nothing

PR submission checklist:

UI Changes testing checklist:
